src/index.css

```
@@ -1,18 +1,112 @@
+  --pf-v6-c-button--m-plain__icon--Color: var(--pf-t--color--white);
+  color: var(--pf-t--color--white);
+  background-color: transparent;
+  border: none;
+  box-shadow: none;
+}
+
+
+/* PF6 pagination page input: match PF5 white background with border */
+.pf-v6-c-page .pf-v6-c-pagination__nav-page-select .pf-v6-c-form-control {
+  --pf-v6-c-form-control--BackgroundColor: var(--pf-t--color--white);
+  background-color: var(--pf-t--color--white);
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)